### PR TITLE
Add tests for YoutubeNode.

### PR DESCRIPTION
**Pull request by cassierecher**

### CODE REVIEW

File: claat/nodes/youtube_test.go [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+			in:   "Mlk888FiI8A",
+			out: &YouTubeNode{
+				node:    node{typ: NodeYouTube},
+				VideoID: "Mlk888FiI8A",
+			},
+		},
+	}
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			out := NewYouTubeNode(tc.in)
```
Comment (DragonRider0o0) [inline claat/nodes/youtube_test.go:32]: Consider renaming `in` to `videoID`

File: claat/nodes/youtube_test.go [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+				return
+			}
+		})
+	}
+}
+
+func TestYouTubeNodeEmpty(t *testing.T) {
+	tests := []struct {
+		name string
+		in   *YouTubeNode
```
Comment (DragonRider0o0) [inline claat/nodes/youtube_test.go:44]: Use the same pattern for both tests, either pass the video ID string in both or pass the YouTubeNode in both.

Comment (cassierecher) [reply claat/nodes/youtube_test.go:44]: Discussion on similar comment https://github.com/googlecodelabs/tools/pull/596/files/591b1ac68607383d635229cd063ed8561fc4b93d

File: claat/nodes/youtube_test.go (hunk 1 above)
Comment (cassierecher) [reply claat/nodes/youtube_test.go:32]: Discussion on similar comment https://github.com/googlecodelabs/tools/pull/596/files/591b1ac68607383d635229cd063ed8561fc4b93d

**Review by shawnbuso — APPROVED** [review]

**Pull request merged**